core/src/main/scala/chisel3/reflect/DataMirror.scala

```diff
@@ -194,11 +194,66 @@ object DataMirror {
     }
   }
 
-  /** Return all expanded components, including intermediate aggregate nodes
+  // Old definition of collectLeafChildren
+  @deprecated("Use DataMirror.collectLeafChildren instead")
+  def getLeafs(d: Data): Seq[Data] = collectLeafChildren(d)
+
+  // Old definition of collectAllChildren
+  @deprecated("Use DataMirror.collectAllChildren instead")
+  def getIntermediateAndLeafs(d: Data): Seq[Data] = collectAllChildren(d)
+
+  /** Collect just the leaf components of a data component's children (i.e. anything that isn't
+    * a `Record` or a `Vec`, but an `Element`)
+    *
+    * @param d Data component to recursively collect leaf components.
+    *
+    * @return All `Element` components; intermediate fields/indices are not included
+    */
+  def collectLeafChildren(d: Data): Seq[Data] =
```

Comment:
to me a leaf is not a "child", because a child is only one level down... but maybe thats just me? what was wrong with the old name?